src/Tizen.NUI/src/public/InputMethodContext.cs

```diff
@@ -1102,6 +1104,8 @@ public class CallbackData : Disposable
             /// The state if it owns memory
             /// </summary>
             /// <since_tizen> 5 </since_tizen>
+            [Obsolete("This has been deprecated in API9 and will be removed in API11. Please use SwigCMemOwn which is declared in parent class")]
```

Comment:
Please comment out this change on DevelNUI branch as ACR is required on Master branch.

Reply:
Yes. you are right. 
it has been commented out.
thank you.